birdhouse/optional-components/stac-public-access/config/stac-data-proxy/permissions.cfg.template

```diff
@@ -0,0 +1,10 @@
+# NOTE:
+# Assume 'secure-data-proxy' would exist if needed.
+# Since 'secure-data-proxy' < 'stac-data-proxy-public', it should be loaded first.
```

Comment:
Do you mean 'stac-public-access' not 'stac-data-proxy-public'

Reply:
It is `stac-data-proxy-public` since this is the mapped named in `birdhouse/optional-components/stac-public-access/config/stac-data-proxy/docker-compose-extra.yml` that Magpie will see when listing files.